jib-core/src/integration-test/java/com/google/cloud/tools/jib/builder/BuildStepsIntegrationTest.java

```diff
@@ -216,7 +216,7 @@ public void testSteps_forBuildToDockerDaemon()
                 ImageReference.of("gcr.io", "distroless/java", "latest"),
                 ImageReference.of(null, imageReference, null))
             .build();
-    BuildSteps.forBuildToDockerDaemon(DockerClient.newClient(), buildConfiguration).run();
+    BuildSteps.forBuildToDockerDaemon(new DockerClient.Builder().build(), buildConfiguration).run();
```

Comment:
I'd recommend having a static factory for the fully-default case (like `DockerClient.newDefaultClient()`)